blaze/compute/sql.py

```diff
@@ -111,6 +111,16 @@ def compute_up(t, s, **kwargs):
     return compute(t._scalar_expr, d, post_compute=False).label(name)
 
 
+@dispatch(Concat, (Select, Selectable), (Select, Selectable))
+def compute_up(t, lhs, rhs, **kwargs):
+    if t.axis != 0:
+        raise NotImplementedError(
+            'Cannot concat along a non-zero axis in sql',
```

Comment:
I also think a `ValueError` is better here. Whenever I see `NotImplementedError` I have hope that one day the world will improve. In this case, I don't think we'll ever implement `axis=1` for SQL.